### PR TITLE
Add Standard Error to Weight Template

### PR DESCRIPTION
**Pull request by shawntabrizi**
Fixes https://github.com/paritytech/substrate/issues/7648

This PR extracts the Standard Error from the regression model in the Weight analysis and places that as a comment in the weight files.

### CODE REVIEW

**Review by thiolliere — APPROVED** [review]
awsome thanks

**Comment by shawntabrizi** [comment]
bot merge

**Comment by shawntabrizi** [comment]
bot merge

**Review by bkchr — APPROVED** [review]

File: utils/frame/benchmarking-cli/src/writer.rs [hunk 1]
```diff
@@ -145,27 +147,45 @@ fn get_benchmark_data(batch: &BenchmarkBatch) -> BenchmarkData {
...
-		if !slope.is_zero() {
-			if !used_components.contains(&name) { used_components.push(name); }
-			used_writes.push(ComponentSlope { name: name.clone(), slope });
-		}
-	});
+	extrinsic_time.slopes.into_iter()
+		.zip(extrinsic_time.names.iter())
+		.zip(extrinsic_time.model.unwrap().se.regressor_values.iter())
+		.for_each(|((slope, name), error)| {
+			if !slope.is_zero() {
```
Comment (bkchr) [inline utils/frame/benchmarking-cli/src/writer.rs:154]: Ahh fuck :P  This could have been a `filter()`. But yeah, not that important xD

Comment (shawntabrizi) [reply utils/frame/benchmarking-cli/src/writer.rs:154]: Would it actually be better given that i need to use all the variables in the code block below?

I would need a `filter` statement on `slope` and a `then` statement looking at all the variables again?

Comment (bkchr) [reply utils/frame/benchmarking-cli/src/writer.rs:154]: You would just remove the if. Nothing else would change.

**Pull request merged**